app/serializers/order.js

```diff
@@ -37,7 +37,7 @@ export default ApplicationSerializer.extend(CustomPrimaryKeyMixin, {
     const json = this._super(...arguments);
     let attendeeSnapshots = snapshot._hasManyRelationships.attendees;
     let attendees = { data: [] };
-    attendeeSnapshots.forEach(function(snapshot) {
+    attendeeSnapshots && attendeeSnapshots.forEach(function(snapshot) {
```

Comment:
Hi thanks for the PR! 
What does this do?

Reply:
@YogeshSharma0201 Ping

Reply:
@CosmicCoder96 it adds that 'tick' sign on top of 'cross' sign for orders which are placed but not completed, so that when completed the organizers can mark that order as completed.

Reply:
@CosmicCoder96 It checks if attendeeSnapshots is not undefined or null before iterating over it, i was encountering error related to it.

Reply:
If you want to make a check, ensure its explicit via an `if` statement. Even though this syntax gets the job done, it isn't very readable.